### PR TITLE
Flag for stopping active learning when num transitions collected

### PR DESCRIPTION
**Pull request by amburger66**

### CODE REVIEW

**Review by tomsilver — APPROVED** [review]
Lgtm after unit test!

**Review by ronuchit — COMMENTED** [review]
rename to something more descrptive like online_learning_max_transitions?

File: tests/test_main.py [hunk 1]
```diff
@@ -110,6 +110,17 @@ def test_main():
         "--num_test_tasks", "3", "--predicate_mlp_classifier_max_itr", "100"
     ]
     main()
+    # Cover the case where online learning stops after collecting max number of
+    # transitions.
+    sys.argv = [
```
Comment (tomsilver) [inline tests/test_main.py:115]: how long does this test take to run?

Comment (amburger66) [reply tests/test_main.py:115]: on my computer, `test_main` went from 6.4s to 12.6s :/

Comment (ronuchit) [reply tests/test_main.py:115]: why not just lump this in with the test on the previous line?

**Review by tomsilver — APPROVED** [review]

**Pull request merged**